utils/docker/Dockerfile.centos.7

```diff
@@ -79,7 +79,8 @@ RUN yum -y install \
 #  - excluding mpi4py as it depends on CORCI-635
 RUN pip3 install avocado-framework; \
     pip3 install avocado-framework-plugin-result-html; \
-    pip3 install avocado-framework-plugin-varianter-yaml-to-mux
+    pip3 install avocado-framework-plugin-varianter-yaml-to-mux; \
+    pip3 install codespell
```

Comment:
Can you open a ticket to create a codespell RPM.  It's already packaged for Fedora so an EL7 package should be trivial.

Probably, really, we should be doing our linting, etc. on Fedora anyway.  It's tools are typically newer.

Reply:
Is that necessary?  I think the Docker stuff was fairly efficient at caching steps so what would a RPM buy us?

Reply:
It buys us predictability and reproducibility.  `pip install` does not install any particular version and will always just install the latest.  Installing RPMs installs a known/tested version or at minimum documents the version being installed.

Reply:
True.  I'll file a ticket then.

Reply:
Can you just let me know the ticket number when you do so that I can make sure it's on our radar?